x-pack/test/ui_capabilities/security_and_spaces/tests/nav_links.ts

```diff
@@ -62,6 +62,7 @@ export default function navLinksTests({ getService }: FtrProviderContext) {
             expect(uiCapabilities.value).to.have.property('navLinks');
             expect(uiCapabilities.value!.navLinks).to.eql(navLinksBuilder.only('management'));
             break;
+          // if we don't have access at the space itself, security interceptor responds with 404.
```

Comment:
__note:__ I'd like to figure out if we're okay with the change in behavior or we should find a workaround. The reason for the change here is that now Security plugin registers `postAuth` interceptor before Space's one.

Reply:
I'm fine with the change in behavior. I think it was potentially a better user-experience previously, but I don't think it's worth the effort for us to invest too much effort to keep it.